govcd/system.go

```diff
@@ -232,3 +232,42 @@ func getExtension(client *Client) (*types.Extension, error) {
 
 	return extensions, err
 }
+
+// QueryProviderVdcStorageProfileByName finds a provider VDC storage profiles by name
```

Comment:
```suggestion
// QueryProviderVdcStorageProfileByName finds a provider VDC storage profile by name
```

Reply:
Changed. Just FYI using these queries by name we can receive a few results.. If the name would some "*" or not full name. It depends on vCD how it will interprate.